src/core/core.legend.js

```diff
@@ -52,6 +52,7 @@ module.exports = function(Chart) {
 						lineJoin: dataset.borderJoinStyle,
 						lineWidth: dataset.borderWidth,
 						strokeStyle: dataset.borderColor,
+						pointStyle: dataset.pointStyle,
```

Comment:
We should document this in `Legend Item Interface` section of 01-Chart-Configuration.md